src/tests/ftest/util/server_utils.py

```diff
@@ -269,10 +280,6 @@ def run_server(test, hostfile, setname, uri_path=None, env_dict=None,
                         ", ".join(
                             [str(result[key]) for key in result if key != 0])))
 
-        # Pile of build time variables
-        with open(os.path.join(test.basepath, ".build_vars.json")) as json_vars:
-            build_vars = json.load(json_vars)
-
```

Comment:
This is needed for more than just determining the shared tmp dir.  The subsequent lines will result in an error.  This should be restored.